### ISSUE TITLE
yarn build core build nothing

### ISSUE THREAD

**Issue by mmarkelov** [facebook/react#14175]
Seems after this https://github.com/facebook/react/commit/051272f2012e1c2f75dc8ee9fe72a1d85cdac302 `yarn build core --type=UMD` is useless now.
So it should be `yarn build react --type=UMD` and [contribution-prerequisites](https://reactjs.org/docs/how-to-contribute.html#contribution-prerequisites) should be updated.

**Comment by gaearon** [comment]
Yeah, it just changed. Can you please update it to say `yarn build react/index react-dom/index --type=UMD`?

Also same needs to be done in `fixtures/unstable-async/` readmes.

**Comment by zhang14725804** [comment]
download react-16.8.0 source code and run "yarn"

follow Development Workflow

run "yarn build react/index,react-dom/index --type=UMD"

but react-16.8.0/build/node_modules/react/umd/react.development.js net::ERR_FILE_NOT_FOUND

build folder only contains facebook-www folder and react-native folder

**Comment by zhang14725804** [comment]
@gaearon @mmarkelov
